spock-core/src/main/java/org/spockframework/util/MopUtil.java

```diff
@@ -25,8 +25,7 @@
 
 public abstract class MopUtil {
   private static final Field ReflectionMetaMethod_method  = getDeclaredField(ReflectionMetaMethod.class, "method");
-  private static final Field CachedField_field = GroovyRuntimeUtil.isGroovy2() ? getDeclaredField(CachedField.class, "field") :
-    getDeclaredField(CachedField.class, "cachedField");
+  private static final Field CachedField_field = getDeclaredField(CachedField.class, "field");
```

Comment:
was this fixed with groovy 3.0.5?

Reply:
Depends on how you define "fixed".
Up to 3.0.5 it is `cachedField`.
In 3.0.6 they renamed it back to `field`.